src/Pos/Worker.hs

```diff
@@ -58,6 +60,12 @@ allWorkers NodeResources {..} = mconcatPair
     , wrap' "delegation" $ dlgWorkers
     , wrap' "slotting"   $ (properSlottingWorkers, mempty)
 
+    , case ncTopology ncNetworkConfig of
+        TopologyBehindNAT dnsDomains ->
+          wrap' "subscription" $ subscriptionWorkers ncNetworkConfig dnsDomains
+        _otherwise ->
+          mempty -- No subscription worker required
```

Comment:
Hmmm just realized that for the static core/relay nodes we currently don't initiate Kademlia, so running this now gives "internal error: missing dht instance".